gapis/api/transform2/transform_chain.go

```diff
@@ -105,29 +123,35 @@ func (chain *TransformChain) endChain(ctx context.Context) error {
 	return nil
 }
 
-func (chain *TransformChain) transformCommand(ctx context.Context, inputCmd api.Cmd, id api.CmdID) error {
-	var err error
-	cmds := []api.Cmd{inputCmd}
+func (chain *TransformChain) transformCommands(ctx context.Context, id api.CmdID, inputCmds []api.Cmd, beginTransformIndex int) error {
+	for i, transform := range chain.transforms {
```

Comment:
Could this be rewritten with an explicit iteration variable to be shorter and not need to "continue" over the first "beginTransformIndex" commands?

Reply:
If it matters yes we can but I would suggest to adding it to the next PR so that we don't wait for a rebuild. I don't think it is very important.